This patch makes it possible to build KDMThemeManager on systems with
automake != 1.7

--- kdmtheme-1.2.2/configure	2007-09-28 14:48:14.000000000 +0200
+++ kdmtheme-1.2.2/configure.new	2008-01-31 12:45:53.000000000 +0100
@@ -1934,13 +1934,13 @@
 
 # Some tools Automake needs.
 
-ACLOCAL=${ACLOCAL-"${am_missing_run}aclocal-${am__api_version}"}
+ACLOCAL=${ACLOCAL-"${am_missing_run}aclocal"}
 
 
 AUTOCONF=${AUTOCONF-"${am_missing_run}autoconf"}
 
 
-AUTOMAKE=${AUTOMAKE-"${am_missing_run}automake-${am__api_version}"}
+AUTOMAKE=${AUTOMAKE-"${am_missing_run}automake"}
 
 
 AUTOHEADER=${AUTOHEADER-"${am_missing_run}autoheader"}
